src/components/OrderTable/index.tsx

```diff
@@ -3,22 +3,34 @@ import { useDispatch } from 'react-redux';
 import { editStatus } from '../../actions';
 import { Table, Divider, Tag, Select, Button } from 'antd';
 
-function OrderTable({ data }) {
+
+interface Props {
+  data: {
+    key: '1',
+    firstName: 'John',
+    lastName: 'Brown',
+    tableNo: 32,
+    order: 'Rice and jollof',
+    state: 'Jotted down',
+  }[];
+}
+
+function OrderTable({ data }: Props) {
   const dispatch = useDispatch();
-  const [changeState, setChangeState] = useState({ key: '', status: '' })
+  const [changeState, setChangeState] = useState({ key: 0, status: '' })
```

Comment:
You can really take advantage of typescript and autocomplete here as `useState` is a generic function. Which means you can tell the compiler how your state is structured. E.g.

```ts
interface IOrderTableState {
  key: number;
  status: string;
}

const [changeState, setChangeState] = useState<IOrderTableState>({ key: 0, status: '' });

```